bang/magicollections.py

```diff
@@ -2636,6 +2636,18 @@ def to_fields(self, view, item, extra_fields=None, exclude_fields=None, *args, *
                     'image': item.member.square_image_url,
                 })
             extra_fields.append(('costume', member_field_params))
+        
+        chibis_set = item.owned_chibis.all()
```

Comment:
Same comment as above, you can use `prefetch_related` line 2592. It makes even more sense in the list view because it will do a single query for all the costumes instead of one per costume.